README.md

```diff
@@ -877,6 +878,65 @@ Data type: `String`
 
 Name. Made up of access_list and the entry with an underscore seperator. eg. list42_10 is from access_list list42 and entry 10.
 
+#### ios_network_trunk
+
+Ethernet logical (switch-port) interface.  Configures VLAN trunking. Extension of network_trunk.
+
+
+##### Properties
+
+The following properties are available in the `ios_network_trunk` type.
+
+###### `ensure`
+
+Valid values: present, absent.
+
+The basic property that the resource should be in.
+
+Default value: present.
+
+###### `encapsulation`
+
+Valid values: dot1q, isl, negotiate, none.
+
+The vlan-tagging encapsulation protocol, usually dot1q.
+
+###### `mode`
+
+Valid values: access, trunk, dynamic_auto, dynamic_desirable.
+
+The L2 interface mode, enables or disables trunking.
+
+###### `untagged_vlan`
+
+VLAN used for untagged VLAN traffic. a.k.a Native VLAN.
+
+###### `tagged_vlans`
+
+Array of VLAN names used for tagged packets.
+
+###### `pruned_vlans`
+
+Array of VLAN ID numbers used for VLAN pruning.
+
+###### `name`
+
+namevar
+
+The switch interface name, for example, Ethernet1.
+
+###### `access_vlan`
+
+The VLAN the access port is to use when set in access mode.
+
+###### `allowed_vlans`
+
+An enumerated or string value, or a tuple consisting of both, that is used to designate which VLANs can be used when trunking.
```

Comment:
This could do with an example or two.

Reply:
These docs are not following the correct format. The need to display the data `type` as described in the Type and also so the `desc`.

Reply:
Format updated and examples added